src/libraries/EthFlowOrder.sol

```diff
@@ -47,14 +59,16 @@ library EthFlowOrder {
         returns (GPv2Order.Data memory)
     {
         if (order.receiver == GPv2Order.RECEIVER_SAME_AS_OWNER) {
-            // The receiver field specified which address is going to receive the proceeds from the orders. If using
+            // The receiver field specified which address is going to receive the proceeds from the oders. If using
```

Comment:
Is this intended?

Reply:
Thanks for spotting, this most likely comes from a rebase from [this PR](https://github.com/cowprotocol/ethflowcontract/pull/5) and I missed out a commit. It's the only extra commit, so there shouldn't be other issues with this.